src/hooks/airdrop/useWebsocket.ts

```diff
@@ -7,6 +7,8 @@ import { invoke } from '@tauri-apps/api/tauri';
 
 let socket: ReturnType<typeof io> | null;
 
+const NETWORK = 'esmeralda';
```

Comment:
That's not going to work on nextnet

Reply:
Hey yeah this is for testing purposes won't go live yet